panel/src/components/Forms/Login.vue

```diff
@@ -1,58 +1,84 @@
+      <h1 class="k-offscreen">{{ $t('login') }}</h1>
 
-    <div v-if="issue" class="k-login-alert" @click="issue = null">
-      <span>{{ issue }}</span>
-      <k-icon type="alert" />
-    </div>
+      <div v-if="issue" class="k-login-alert" @click="issue = null">
+        <span>{{ issue }}</span>
+        <k-icon type="alert" />
+      </div>
 
-    <div class="k-login-fields">
-      <button
-        v-if="canToggle === true"
-        class="k-login-toggler"
-        type="button"
-        @click="toggleForm"
-      >
-        {{ toggleText }}
-      </button>
-
-      <k-fieldset
-        ref="fieldset"
-        v-model="user"
-        :novalidate="true"
-        :fields="fields"
-      />
-    </div>
+      <div class="k-login-fields">
+        <button
+          v-if="canToggle === true"
+          class="k-login-toggler"
+          type="button"
+          @click="toggleForm"
+        >
+          {{ toggleText }}
+        </button>
 
-    <div class="k-login-buttons">
-      <span
-        v-if="isResetForm === false"
-        class="k-login-checkbox"
-      >
-        <k-checkbox-input
-          :value="user.remember"
-          :label="$t('login.remember')"
-          @input="user.remember = $event"
+        <k-fieldset
+          ref="fieldset"
+          v-model="user"
+          :novalidate="true"
+          :fields="fields"
         />
-      </span>
+      </div>
+
+      <div class="k-login-buttons">
+        <span
+          v-if="isResetForm === false"
+          class="k-login-checkbox"
+        >
+          <k-checkbox-input
+            :value="user.remember"
+            :label="$t('login.remember')"
+            @input="user.remember = $event"
+          />
+        </span>
+        <k-button
+          class="k-login-button"
+          icon="check"
+          type="submit"
+        >
+          {{ $t("login" + (isResetForm ? ".reset" : "")) }}
+          <template v-if="isLoading">…</template>
+        </k-button>
+      </div>
+    </form>
+
+    <div v-if="Object.keys(methods).length > 1" class="k-login-view-buttons">
       <k-button
-        class="k-login-button"
-        icon="check"
-        type="submit"
-      >
-        {{ $t("login" + (isResetForm ? ".reset" : "")) }}
-        <template v-if="isLoading">…</template>
-      </k-button>
+        v-for="(method, methodName) in methods"
+        :key="methodName"
+        :current="view === methodName"
+        :icon="method.icon"
+        :tooltip="method.label"
+        @click="triggerMethod(methodName)"
+      />
     </div>
-  </form>
+  </div>
 </template>
 
 <script>
+let pluginMethods = {}, plugins = {};
+for (const methodName in window.panel.plugins.loginMethods) {
+  pluginMethods[methodName] = window.panel.plugins.loginMethods[methodName];
+
+  if (pluginMethods[methodName].component) {
+    plugins["k-plugin-login-" + methodName] = pluginMethods[methodName].component;
+    pluginMethods[methodName].component = "k-plugin-login-" + methodName;
```

Comment:
For consistency with https://github.com/getkirby/kirby/pull/2929/files/2eb19a03d25c20b923076baf7b8b4304ff9c0bfa#r527535315 we should probably rename the component name to `"k-login-" + methodName + "-plugin"` as well, right?

Reply:
Yes, that would be great.

Reply:
@bastianallgeier Done. ✅